### PR TITLE
KIALI-2970 Removes fixed height of 95vh for page

### PR DESCRIPTION
**Pull request by josejulio**
** Describe the change **

Patternfly sets the height of the "Page" to 95vh, so we can our calc wrong in the GraphPage.
Additionally, sets the header height to the remaining 5vh, if not, when the screen is big (height about 1800px) a black bar appears on bottom.

### CODE REVIEW

**Comment by israel-hdez** [comment]
Hmm.... But on "normal" screens, it runs out of the viewport:

image

**Comment by josejulio** [comment]
Ah, I must have something wrong in the calcs. I though i tested it on normal screens too :)

**Comment by josejulio** [comment]
Ah yes, the topbar is bigger than 5vh in normal screen. The problem is that the page (defined by PF) is 95vh, I'll check this again later.

File: src/components/Nav/Navigation.tsx [hunk 1]
```diff
@@ -107,9 +113,9 @@ class Navigation extends React.Component<PropsType, NavigationState> {
     );
 
     return (
-      <Page header={Header} sidebar={Sidebar} onPageResize={this.onPageResize}>
+      <Page style={{ backgroundColor: 'purple' }} header={Header} sidebar={Sidebar} onPageResize={this.onPageResize}>
```
Comment (israel-hdez) [inline src/components/Nav/Navigation.tsx]: Just for curiosity, why purple?

Comment (josejulio) [reply src/components/Nav/Navigation.tsx]: Debugging purpose :) 
Was to easily see if my Page filled the whole page.

Comment (josejulio) [reply src/components/Nav/Navigation.tsx]: Will remove this

Comment (josejulio) [reply src/components/Nav/Navigation.tsx]: I just removed this purple color

Comment (israel-hdez) [reply src/components/Nav/Navigation.tsx]: Ok, looks better now :)

**Comment by josejulio** [comment]
@mtho11 if you were still having the issue with the graph toolbar (legend, fit, etc) being hidden on chrome, could you check if this PR helps?

File: src/components/Nav/Navigation.tsx (hunk 1 above)
Comment (mtho11) [reply src/components/Nav/Navigation.tsx]: Purple, haha, I do pink.

**Comment by mtho11** [comment]
The legend fit issues are still there on OS/X laptop:

overlap

But it is not any worse, so I'm not blocking or endorsing. Either way, I'm good.

**Pull request merged**